### PR TITLE
fix(examplebroker): Improve wording for the regenerate code button label

### PR DESCRIPTION
**Pull request by 3v1n0**
The user is allowed to regenerate the login code not the QR code, so
  improve the wording here.
  
  Regenerate the golden files based on that

This is not strictly needed, but it's better to align with https://github.com/ubuntu/authd-oidc-brokers/pull/47

### CODE REVIEW

File: pam/integration-tests/testdata/tapes/cli/qr_code.tape [hunk 1]
```diff
@@ -4,7 +4,7 @@ Output qr_code.gif # If we don't specify a .gif output, it will create a default
 # Configuration header to standardize the output.
 # Does not work with the "Source" command.
 Set Width 800
-Set Height 600
```
Comment (denisonbarbosa) [inline pam/integration-tests/testdata/tapes/cli/qr_code.tape:7]: Why was this necessary? We only have a word change, we should not need to increase the height.

Comment (3v1n0) [reply pam/integration-tests/testdata/tapes/cli/qr_code.tape:7]: See this commit: https://github.com/ubuntu/authd/pull/408#pullrequestreview-2154767767

Comment (3v1n0) [reply pam/integration-tests/testdata/tapes/cli/qr_code.tape:7]: Ouch, wrong paste: https://github.com/ubuntu/authd/pull/408/commits/37f6d560e115a85a78495a18f56fa5629b139f73

**Review by denisonbarbosa — APPROVED** [review]
Looks good to me. Nicely done!

File: pam/integration-tests/testdata/TestCLIAuthenticate/golden/authenticate_user_with_qr_code_in_a_tty [hunk 2]
```diff
@@ -276,6 +294,9 @@ Scan the qrcode or enter the code in the login page
 ██████████████████████████████████████████████████████████████████
                                1337
 
+PAM Authenticate() for user "user-integration-qr-code-tty" exited with success
```
[the remark this answers is not in the document]
Comment (3v1n0) [reply pam/integration-tests/testdata/TestCLIAuthenticate/golden/authenticate_user_with_qr_code_in_a_tty:297]: Yeah, I think due to the lack of vertical space because of the bigger qrcode.

Sadly we didn't catch it in #393, but I noticed it here since I saw that changing the button label wasn't respected by these golden files. And it was just because there was not enough vertical space for the button and error messages to show, as written on the [commit](https://github.com/ubuntu/authd/commit/37f6d560e115a85a78495a18f56fa5629b139f73) that this change belongs to.

[the remark this answers is not in the document]
Comment (didrocks) [reply pam/integration-tests/testdata/TestCLIAuthenticate/golden/authenticate_user_with_qr_code_in_a_tty:297]: ack then!

**Review by didrocks — APPROVED** [review]
